src/Analyzers/Analyzers/src/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1 @@
+#nullable enable
```

Comment:
@mavasani Regarding this, how much does it matter whether we've actually started annotating these libraries here?

Reply:
@jcouv would be a better person to answer this.

Reply:
Based on the lack of errors about this, I suggest we include `#nullable enable` everywhere. Makes life simpler.